synapse/lib/stormlib/json.py

```diff
@@ -45,3 +121,14 @@ async def _jsonLoad(self, text):
         except Exception as e:
             mesg = f'Text is not valid JSON: {text}'
             raise s_exc.BadJsonText(mesg=mesg)
+
+    async def _jsonSchema(self, schema):
+        schema = await s_stormtypes.toprim(schema)
+        # We have to ensure that we have a valid schema for making the object.
+        try:
+            await s_coro.forked(compileJsSchema, schema)
```

Comment:
This could block storm parsing since the default pool size is 1.  Could call s_coro.spawn() like the xml lib.